java/ql/src/experimental/Security/CWE/CWE-089/MyBatisAnnotationSqlInjectionLib.qll

```diff
@@ -0,0 +1,128 @@
+/**
+ * Provides classes for SQL injection detection in MyBatis annotation.
+ */
+
+import java
+import MyBatisCommonLib
+import semmle.code.xml.MyBatisMapperXML
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.frameworks.Properties
+
+/** A sink for MyBatis annotation method call an argument. */
+class MyBatisAnnotationMethodCallAnArgument extends DataFlow::Node {
```

Comment:
```suggestion
/** An argument of a MyBatis annotated method. */
class MyBatisAnnotatedMethodCallArgument extends DataFlow::Node {
```
Name and document classes for what they are, not how you intend to use them

Reply:
Get.